--- xShip/kinox.py
+++ xVault/kinox.py
@@ -1,5 +1,5 @@
 # -*- coding: UTF-8 -*-
-from resources.lib.utils import isBlockedHosterFast as isBlockedHoster
+from resources.lib.utils import isBlockedHoster
 import json
 import re
 import time
@@ -197,7 +197,7 @@
                             r = [(dom_parser.parse_dom(r, 'a', req='href'), dom_parser.parse_dom(r, 'iframe', req='src'))]
                             r = [i[0][0].attrs['href'] if i[0] else i[1][0].attrs['src'] for i in r if i[0] or i[1]][0]
                             if not r.startswith('http'): r = urljoin('https:', r)
-                            isBlocked, hoster, url, prioHoster = isBlockedHoster(r)
+                            isBlocked, hoster, url, prioHoster = isBlockedHoster(r, isResolve=False)
                             if isBlocked: continue
                             info = 'Mirror ' + str(x+1)
                             if language_label:
